brave/brave-ios
# Fix #3022: Update Yandex search engine.

## Summary of Changes


This pull request fixes #3022 

## Submitter Checklist:


## Test Plan:

This change goes to 10 different regions, I think we do not test them all, we can verify with Russian region as a start.
Server side verification should not be needed to at the moment, we can revisit after 1.22 ships.

New User path, default search engine verification:
- Set device region to Russia
- Fresh install 1.22 build
- Verify that Yandex is showing as a first option during onboarding and is selected at start.
- Search for `cats`
- Verify that url goes like this `https://yandex.ru/search/touch/?text=cats&clid=2423859&lr=104797&redircnt=1605785170.1`,
some params might differ in your search, only make sure that `text` and `clid` param are correct
- Enable search suggestions
- Verify that search suggestions are showing

Existing User path, do not overwrite default search engine verification:
- Set device region to Russia
- Fresh install 1.21 build, set Google as a default engine
- Upgrade to 1.22
- Make a search query, verify that Google is still used as a default engine
- Go to app settings, switch search engine to Yandex
- Search for `cats`
- Verify that url goes like this `https://yandex.ru/search/touch/?text=cats&clid=2423859&lr=104797&redircnt=1605785170.1`,
some params might differ in your search, only make sure that `text` and `clid` param are correct
- Enable search suggestions
- Verify that search suggestions are showing

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
